lib/article_json/article.rb

```diff
@@ -91,10 +91,15 @@ def to_plain_text
     # article. If the method is called multiple times, the order of additional
     # elements is maintained.
     # @param [Object] additional_elements
-    def place_additional_elements(additional_elements)
+    # @param [ArticleJSON::Utils] with
```

Comment:
I am unsure how to define here the param type :thinking: @Devex/back-end-guild

Reply:
I think you need to define a duck type here (walks like a duck, quacks like a duck, it probably is a duck!).
So, to say that you expect a class that responds to `#merge_elements`, you'd do this:
```suggestion
    # @param [Class<#merge_elements>] with
```
This still doesn't cover the expected signature for `#initialize`, but you can specify this in the free-text comment, that should be good enough...

Reply:
let's see if that looks better